charts/loki-stack/requirements.yaml

```diff
@@ -0,0 +1,29 @@
+dependencies:
+- name: "loki"
+  condition: loki.enabled
+  repository: "file://../loki"
+  version: "^0.32.0"
+- name: "promtail"
+  condition: promtail.enabled
+  repository: "file://../promtail"
+  version: "^0.26.0"
+- name: "fluent-bit"
+  condition: fluent-bit.enabled
+  repository: "file://../fluent-bit"
+  version: "^0.4.0"
+- name: "grafana"
+  condition: grafana.enabled
+  version: "~3.8.15"
+  repository:  "https://kubernetes-charts.storage.googleapis.com/"
+- name: "prometheus"
+  condition: prometheus.enabled
+  version: "~9.3.0"
+  repository:  "https://kubernetes-charts.storage.googleapis.com/"
+- name: "filebeat"
+  condition: filebeat.enabled
+  version: "~7.8.0"
+  repository:  "https://helm.elastic.co"
+- name: "logstash"
+  condition: logstash.enabled
+  version: "~2.4.0"
+  repository:  "https://kubernetes-charts.storage.googleapis.com/"
```

Comment:
There is a logstash chart in the elastic charts repo https://github.com/elastic/helm-charts#charts, but the stable chart hasn't officially deprecated in it's favor yet. You might want to make an issue here to track that (part of that issue would also be cleaning up the stable URL from `chart-repos` in `ct.yaml`.

Any help with getting the rest of these elastic charts officially deprecated in stable would be appreciated.